third_party/move/move-vm/transactional-tests/tests/paranoid-tests/encapsulation_safety/cross_module_pack_closure.exp

```diff
@@ -0,0 +1,46 @@
+processed 13 tasks
+
+task 4 'run'. lines 54-54:
+Error: Function execution failed with VMError: {
+    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
```

Comment:
Is this the standard issue with error propagation?

Reply:
Wdym by standard issue? The error returned on incorrect visibility is invariant violation, but we set substatus to 1 to indicate paranoid failure.